server/signup.php

```diff
@@ -4,39 +4,42 @@ function getLitmusQuestions() {
 	require 'config.php';
 	$conn = $DB_CONN;
 
-	$queryStatement = $conn->prepare('SELECT lq.litmusQuestionId, pa.possibleAnswerId, pa.text AS possibleAnswerText,
-				lq.text AS litmusQuestionText, lq.required, lq.lookupName
-				FROM PossibleAnswer pa
-				JOIN LitmusQuestion lq ON pa.litmusQuestionId = lq.litmusQuestionId
-				WHERE lq.archived = FALSE AND pa.archived = FALSE
-				ORDER BY lq.orderIndex, pa.orderIndex');
+	# smile and wave boys, this will all be getting replaced
```

Comment:
Most of this file will be deleted once the front-end gets hard-coded. So, just commenting this out until someone gets to it.